src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -209,13 +226,13 @@ cdef class Face:
         if self._hb_face is not NULL:
             hb_face_destroy(self._hb_face)
 
-    """ use bytes/bytearray, not Blob
     @classmethod
-    def create(self, blob: Blob, index: int):
+    def create(cls, fontdata: bytes, index: int):
```

Comment:
since it's a python function, here we could probably accept only just bytes, but a union of bytes/bytearray/array.array or anything that exposes the buffer interface really.

Reply:
I meant _not_ only

Reply:
so you're looking for _memoryview_? also can you keep the name blob (and below hb_blob), thus far I keep the same signature names as harfbuzz. (and bytes is a binary "blob" so.. we can revisit later if needed)